docs/backends/geoip.rst

```diff
@@ -164,6 +164,7 @@ Keys explained
                            format (e.g. %cc).
   :custom_mapping: Defines the mapping between the lookup format and a custom value to replace ``%mp`` placeholder.
 
+:zones_dir: Directory to load zones from. Each file must contain key zone which contains a domain as above.
```

Comment:
I had trouble understanding this until I looked at the test.

How about: "each file must contain exactly one ``zone:`` object, formatted like individual domains in the example configuration above" ?